plugins/liip/repaircafe/components/eventlist/default.htm

```diff
@@ -1,63 +1,85 @@
+                    <div class="row">
+                        {% if __SELF__.condensed %}
+                            <div class="col-md-6 mb-3">
+                                <h3 class="h5">{{ event.title ? event.title : event.cafe.title }}</h3>
                                 <p class="small">
                                     {% if event.start|date('d.m.Y') == event.end|date('d.m.Y') %}
-                                        {{ event.start|date('d.m.Y') }}<br />
+                                        {{ event.start|date('d.m.Y') }}
                                     {% else %}
-                                        {{ event.start|date('d.m.Y') }} - {{ event.end|date('d.m.Y') }}<br />
+                                        {{ event.start|date('d.m.Y') }} - {{ event.end|date('d.m.Y') }}
+                                    {% endif %}
+                                    {{ event.start|date('H:s') }} - {{ event.end|date('H:s') }}
+                                    {% if event.street or event.city %}
+                                        {% set address_string = (event.street ? event.street : '') %}
+                                        {% if event.city and event.street %}
+                                            {% set address_string = address_string ~ ', ' %}
+                                        {% endif %}
+                                        {% if event.city %}
+                                            {% set address_string = address_string ~ (event.zip ? event.zip ~ ' ') ~ event.city %}
+                                        {% endif %}
+                                        <br />{{ address_string }}
                                     {% endif %}
-                                    {{ event.start|date('H:s') }} - {{ event.end|date('H:s') }}<br />
-                                    TODO: address
                                 </p>
                             </div>
+                        {% else %}
+                            <div class="col-md-6 media mb-3">
+                                <img class="d-flex mr-3 w-25" src="{{ event.cafe.logo.path }}" alt="{{ event.cafe.title }}">
+                                <div class="media-body">
+                                    <h3 class="h5"><a href="{{ 'cafe'|page({ slug: event.cafe.slug }) }}">{{ event.cafe.title }}{{ event.title ? ': ' ~ event.title }}</a></h3>
+                                    <p class="small">
+                                        {% if event.start|date('d.m.Y') == event.end|date('d.m.Y') %}
+                                            {{ event.start|date('d.m.Y') }}
+                                        {% else %}
+                                            {{ event.start|date('d.m.Y') }} - {{ event.end|date('d.m.Y') }}
+                                        {% endif %}
+                                        <br />{{ event.start|date('H:s') }} - {{ event.end|date('H:s') }}
```

Comment:
same as above: `event.end` is not required so it can be empty.

maybe a partial makes sense for this, or is that not possible here?